geomesa-memory/src/main/java/org/locationtech/geomesa/memory/cqengine/attribute/SimpleFeatureAttribute.java

```diff
@@ -0,0 +1,29 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.memory.cqengine.attribute;
+
+import com.googlecode.cqengine.attribute.SimpleAttribute;
+import com.googlecode.cqengine.query.option.QueryOptions;
+import org.opengis.feature.simple.SimpleFeature;
+
+public class SimpleFeatureAttribute<A> extends SimpleAttribute<SimpleFeature, A> {
```

Comment:
I started it as a fork of CQEngine.  Also, I might be interested in contributing the code to CQEngine or GeoTools, etc.